x-pack/plugins/lens/public/drag_drop/drag_drop.tsx

```diff
@@ -136,17 +148,23 @@ const DragDropInner = React.memo(function DragDropInner(
     setDragging,
     isDragging,
     isNotDroppable,
+    dragType = 'copy',
+    dropType = 'add',
   } = props;
 
   const classes = classNames(
     'lnsDragDrop',
-    className,
     {
+      'lnsDragDrop-isDraggable': draggable,
+      'lnsDragDrop-isDragging': isDragging,
+      'lnsDragDrop-isHidden': isDragging && dragType === 'move',
```

Comment:
🏳️ The one thing I couldn't get working is this class to apply when dragging a configuration. Investigating on the consuming side it seems that `isDragging` never updates to `true` for configs, only for field items. I cannot figure out why.

Reply:
This is happening because in https://github.com/elastic/kibana/pull/78674/files#diff-65f69347e8aa4ab0e15d84b1d99c1687R219 the dragged value is always a new object, but the `isDragging` state is using referencial equality: https://github.com/elastic/kibana/pull/78674/files#diff-a51c98e0d1e258967687533be79acfa1R116

The best solution I can think of is to make it possible to overwrite the equality check with a custom function and overwrite ith with this in the layer panel:
```
function isSameConfiguration(config1: unknown, config2: unknown) {
  return (
    config1 &&
    config2 &&
    typeof config1 === 'object' &&
    typeof config2 === 'object' &&
    config1.columnId === config2.columnId &&
    config1.groupId === config2.groupId &&
    config1.layerId === config2.layerId
  );
}
```

I can modify this PR tomorrow.